doc/src/explanation/modules/physics/vector/kinematics/kinematics.rst

```diff
@@ -297,38 +297,38 @@ define the angular velocity of the body :math:`\mathbf{B}` and velocity of the
 point :math:`O`:
 
 .. math::
-  {}^\mathbf{N}\boldsymbol{\omega}^\mathbf{B} &= u_3 \mathbf{\hat{n}_z} = u_3
+  {}^{\mathbf{N}}\boldsymbol{\omega}^{\mathbf{B}} &= u_3 \mathbf{\hat{n}_z} = u_3
   \mathbf{\hat{b}_z}\\
-  {}^\mathbf{N}\mathbf{v}^O &= u_1 \mathbf{\hat{n}_x} + u_2 \mathbf{\hat{n}_y}\\
+  {}^{\mathbf{N}}\mathbf{v}^O &= u_1 \mathbf{\hat{n}_x} + u_2 \mathbf{\hat{n}_y}\\
 
 and accelerations can be written as:
 
 .. math::
-  {}^\mathbf{N}\boldsymbol{\alpha}^\mathbf{B} &= \dot{u_3} \mathbf{\hat{n}_z} =
-  \dot{u_3} \mathbf{\hat{b}_z}\\
-  {}^\mathbf{N}\mathbf{a}^O &= \dot{u_1} \mathbf{\hat{n}_x} + \dot{u_2}
+  {}^{\mathbf{N}}\boldsymbol{\alpha}^{\mathbf{B}} &= \dot{u}_3 \mathbf{\hat{n}_z} =
+  \dot{u}_3 \mathbf{\hat{b}_z}\\
+  {}^{\mathbf{N}}\mathbf{a}^O &= \dot{u}_1 \mathbf{\hat{n}_x} + \dot{u}_2
   \mathbf{\hat{n}_y}\\
 
 We can use the 2 point theorem to calculate the velocity and acceleration of
 point :math:`P` now.
 
 .. math::
   \mathbf{r}^{OP} &= R \mathbf{\hat{b}_x}\\
-  {}^\mathbf{N}\mathbf{v}^P &={}^\mathbf{N}\mathbf{v}^O +
-  {}^\mathbf{N}\boldsymbol{\omega}^\mathbf{B} \times \mathbf{r}^{OP}\\
-  {}^\mathbf{N}\mathbf{v}^P &= u_1 \mathbf{\hat{n}_x} + u_2 \mathbf{\hat{n}_y}
+  {}^{\mathbf{N}}\mathbf{v}^P &={}^{\mathbf{N}}\mathbf{v}^O +
+  {}^{\mathbf{N}}\boldsymbol{\omega}^{\mathbf{B}} \times \mathbf{r}^{OP}\\
+  {}^{\mathbf{N}}\mathbf{v}^P &= u_1 \mathbf{\hat{n}_x} + u_2 \mathbf{\hat{n}_y}
   + u_3 \mathbf{\hat{b}_z} \times R \mathbf{\hat{b}_x} = u_1
   \mathbf{\hat{n}_x} + u_2 \mathbf{\hat{n}_y} + u_3 R \mathbf{\hat{b}_y}\\
-  {}^{\mathbf{N}}\mathbf{a}^P &={}^\mathbf{N}\mathbf{a}^O +
-  {}^\mathbf{N}\boldsymbol{\alpha}^\mathbf{B} \times \mathbf{r}^{OP} +
-  {}^\mathbf{N}\boldsymbol{\omega}^\mathbf{B} \times
-  (^\mathbf{N}\boldsymbol{\omega}^\mathbf{B} \times \mathbf{r}^{OP})\\
-  {}^{\mathbf{N}}\mathbf{a}^P &= \dot{u_1} \mathbf{\hat{n}_x} + \dot{u_2}
-  \mathbf{\hat{n}_y} + \dot{u_3}\mathbf{\hat{b}_z}\times R \mathbf{\hat{b}_x}
+  {}^{\mathbf{N}}\mathbf{a}^P &={}^{\mathbf{N}}\mathbf{a}^O +
+  {}^{\mathbf{N}}\boldsymbol{\alpha}^{\mathbf{B}} \times \mathbf{r}^{OP} +
+  {}^{\mathbf{N}}\boldsymbol{\omega}^{\mathbf{B}} \times
+  (^{\mathbf{N}}\boldsymbol{\omega}^{\mathbf{B}} \times \mathbf{r}^{OP})\\
+  {}^{\mathbf{N}}\mathbf{a}^P &= \dot{u}_1 \mathbf{\hat{n}_x} + \dot{u}_2
+  \mathbf{\hat{n}_y} + \dot{u}_3\mathbf{\hat{b}_z}\times R \mathbf{\hat{b}_x}
   +u_3\mathbf{\hat{b}_z}\times(u_3\mathbf{\hat{b}_z}\times
   R\mathbf{\hat{b}_x})\\
-  {}^{\mathbf{N}}\mathbf{a}^P &= \dot{u_1} \mathbf{\hat{n}_x} + \dot{u_2}
-  \mathbf{\hat{n}_y} + R\dot{u_3}\mathbf{\hat{b}_y} - R u_3^2
+  {}^{\mathbf{N}}\mathbf{a}^P &= \dot{u}_1 \mathbf{\hat{n}_x} + \dot{u}_2
```

Comment:
Regarding all these replacements of `\dot{<letter>_<subscript>}` by `\dot{<letter>}_{<subscript>}`, I promised myself to check later if really  I was not modifying the semantics here.  I hope no and that the subscripts are really indices.

Reply:
Yes it should be `\dot{q}_1` in all cases.